ArduSub/config.h

```diff
@@ -190,6 +190,12 @@
 # define LOGGING_ENABLED                ENABLED
 #endif
 
+// Statistics
+#ifndef STATS_ENABLED
+ # define STATS_ENABLED ENABLED
```

Comment:
```suggestion
 # define STATS_ENABLED (AP_STATS_ENABLED ? ENABLED : DISABLED)
```

.. otherwise you can have someone turning it off on the custom build server and having it fail to compile because it's being taken out at the library level but not the vehicle level.